frontend/src/components/PriceInterval/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from '@mui/material';
+
+const PriceInterval = ({ price, priceTo, priceType, currency }) => {
```

Comment:
Interval kojarzy się bardziej z czasem - można by dać PriceRange, żeby było czytelniejsze.
Jeśli komponent to PriceInterval to dałbym propsy 'from', 'to', 'type', 'currency' - potem będzie ładniejsze użycie tego (wiadomo że to price więc nie trzeba tego przed każdym propsem pisać)